docs/modules/debug/require-inspector.md

```diff
@@ -0,0 +1,28 @@
+<%( TOC )%>
+
+# Require inspector extension
+
+The Require inspector extension is a debugging extension for the Unity inspector, which helps with visualizing the use of `[Require]` in MonoBehaviour components.
```

Comment:
this needs links to the requireables docs 

inspector needs to be captialized according to Unity docs (everywhere you use it in this doc)